client/src/includes/stimulus.test.js

```diff
@@ -0,0 +1,253 @@
+import { Application, Controller } from '@hotwired/stimulus';
```

Comment:
Is there a reason why we’re not writing those tests in TypeScript if the corresponding new code is written in TypeScript?

Reply:
As per comment on other PR (cannot remember which one) - I understood we write unit tests in JS - we only have a very small subset of tests written in TS (2) vs most of them in JS (50+).

I also think it's easier (developer overhead) to write these quickly in JS and allows you to test things that maybe should not happen in theory in TS but could happen when called externally.

My preference would be for us to be clear that we do want all tests in TS before we start doing this - maybe worth discussing when we work on the UI development guide.

https://docs.google.com/document/d/1Sujt_wgeRvTWMFH4v_Br6z7IpEBRdHq-y-OyOpL-Y6o/edit#

Reply:
As per chat - you are happy with this for now @thibaudcolas